lib/python-beta/src/bailo/bailo.py

```diff
@@ -19,32 +19,27 @@ def get(self, *args, **kwargs):
         return requests.get(*args, **kwargs)
 
 
-class BailoClient:
+class BailoClient():
     def __init__(self, url: str, agent: Agent = Agent()):
         self.url = url.rstrip("/") + "/api"
         self.agent = agent
 
     def create_model(
             self,
-            name: str,
-            description: str,
-            visibility: ModelVisibility
+            model: Model,
```

Comment:
I _think_ the syntax for this is actually better as the old argument list, even though it will differ from the `update_model` method.  Comparison:

```python
# old
client.create_model(
  name="Example Model"
  description="Some Description"
  visibility=ModelVisibility.Public
)

# new
client.create_model(
  model={
    "name": "Example Model",
    "description": "Some Description",
    "visibility": ModelVisibility.Public
  }
)
```

This new method is also bad because the `Model` fields are all optional, which isn't accurate.  So it's weakening the typing on this as well.